.github/workflows/test-reporter.yml

```diff
@@ -11,10 +11,27 @@ on:
 jobs:
   report:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.event == 'pull_request' }}
     steps:
       - uses: elastic/apm-pipeline-library/.github/actions/test-report@current
         with:
           artifact: test-results            # artifact name
           name: JUnit Tests                 # Name of the check run which will be created
-          path: "**/*.xml"                  # Path to test results (inside artifact .zip)
+          path: "**/elastic-serverless-forwarder-junit.xml"                  # Path to test results (inside artifact .zip)
           reporter: java-junit              # Format of test results
+          output-to: step-summary           # Write summary in the PR
+      - uses: dawidd6/action-download-artifact@v2
```

Comment:
NIT:

What do you think of using the standard gh cli instead? Similar to https://github.com/elastic/apm-agent-php/blob/c2e643f76060ea731d54aa29b77109630e9dc497/.github/workflows/snapshoty.yml#L18-L25

Otherwise, let's use a pinned version

Reply:
I was looking into it and found https://github.com/cli/cli/issues/5625

not sure how it works on `apm-agent-php` repo, I could find the dockerfile for docker.elastic.co/observability-ci/snapshoty :)